source/sass/11-base/forms/_forms.scss

```diff
@@ -0,0 +1,16 @@
+.views-exposed-form { // exposed view filters
+  .form-item {
+    margin-bottom: 0;
+    margin-right: .4rem;
+  }
+
+  // search form
+  .search {
```

Comment:
Is this generic enough for gent_base?

Reply:
Don't think so; 90% of the projects have a search page, no? 🤔

Reply:
They should, seeing as all pages must be reachable on "multiple ways".

But is base/forms the correct place?

Reply:
I moved the styling for the search form specifically to a separate molecule, called search.